backend/src/test/java/de/sambam/gamenighttracker/controller/UserControllerTest.java

```diff
@@ -66,15 +102,102 @@ public void getAllGamesFromDbTest() {
 
         //THEN
         assertEquals(getResponse.getStatusCode(), HttpStatus.OK);
-        assertThat(gameList.length, is(2));
+        assertThat(gameList.length, is(3));
         assertThat(List.of(gameList), containsInAnyOrder(
                 Game.builder()
                         .name("Bonfire")
                         .releaseYear("2020")
+                        .gameSessionList(List.of(GameSession.builder()
+                                .id("1")
+                                .sessionState("DONE")
+                                .winnerPlayerId("Mario")
+                                .playerList(List.of(player1, player2))
+                                .build()))
                         .build(),
                 Game.builder()
                         .name("Calico")
                         .releaseYear("2020")
+                        .gameSessionList(List.of(GameSession.builder()
+                                .id("2")
+                                .sessionState("DONE")
+                                .winnerPlayerId("samBAM")
+                                .playerList(List.of(player3, player4))
+                                .build()))
+                        .build(),
+                Game.builder()
+                        .name("Calico")
+                        .releaseYear("2020")
+                        .gameSessionList(List.of(GameSession.builder()
+                                .id("3")
+                                .sessionState("DONE")
+                                .winnerPlayerId("Sanne")
+                                .playerList(List.of(player5, player6))
+                                .build()))
                         .build()));
     }
+
+    @Test
+    @DisplayName("GET request to /user/gamesessions should return all game sessions")
+    public void getGameSessionsListTest() {
+        //WHEN
+        ResponseEntity<GameSession[]> getResponse = testRestTemplate.getForEntity(
+                "http://localhost:" + port + "user/gamesessions", GameSession[].class);
```

Comment:
you will probably need this url in multiple tests, you may want to consider extracting it to a private helper method which you only need to extend with your specific endpoint path
```
private String getUrl() { return "http://localhost:" + port + "/";}
```